docs/index.md

```diff
@@ -51,20 +51,21 @@ Usage:
   dashboard-linter lint [dashboard.json] [flags]
 
 Flags:
-  -c, --config string   path to a configuration file
-      --fix             automatically fix problems if possible
-  -h, --help            help for lint
-      --strict          fail upon linting error or warning
-      --verbose         show more information about linting
+  -c, --config string         path to a configuration file
+      --fix                   automatically fix problems if possible
+  -h, --help                  help for lint
+  -m, --matcher stringArray   matcher required to be present in all selectors, e.g. 'instance=~"$instance"' or 'cluster="$cluster"', can be specified multiple times (default ["instance=~""$instance""","job=~""$job"""])
```

Comment:
I'd like to see this configured in the `.lint` config file rather than as a CLI flag. In the case of linting several dashboards in a repo, or when embedding into something like [mixtool ](https://github.com/monitoring-mixins/mixtool) that makes this a bit more flexible, and can be different per dashboard-project.

Something like;
```
---
settings:
  template-variable-matchers-rule:
    matchers:
    - instance=~""
    - instance=""
    - job=~""
```